Cognite.Simulator.Tests/UtilsTests/SimulationRunnerTest.cs

```diff
@@ -286,11 +286,11 @@ bool checkTs
 
                     // Check that the correct input sample was added as a data point
                     var inDps = await cdf.DataPoints.ListAsync(
-                        new DataPointsQuery
+                        new CogniteSdk.DataPointsQuery
                         {
                             Start = simulationTime.ToString(),
                             End = (simulationTime + 1).ToString(),
-                            Items = inTs.Select(i => new DataPointsQueryItem
+                            Items = inTs.Select(i => new CogniteSdk.DataPointsQueryItem
```

Comment:
if you do this `using DataPointsQuery = CogniteSdk.DataPointsQuery;` on the top of the file, then only one line needs to be changed

Reply:
Fixed :))